wiki/osu!api/ru.md

```diff
@@ -0,0 +1,10 @@
+У osu! есть публичный API, который сторонние сервисы могут использовать для расширения и поддержки osu! платформы. Раньше он был более приватным, использовался только для каждого запроса, но теперь любой может использовать API, если у него есть учетная запись osu!.
```

Comment:
missing title before this, in english it has
```markdown
# osu!api
```

Reply:
oh, sure